docs/replication-manager/replication-manager-for-pxc.md

```diff
@@ -0,0 +1,280 @@
+# Replication manager for Percona XtraDB Cluster
+
+The replication manager script helps manage [multi-source replication](../glossary.md#multi-source-replication) between multiple Percona XtraDB Cluster clusters.
+
+## Preconditions
```

Comment:
Should this be Example?
Preconditions imply that we should tell them what they need beforehand.

Reply:
Updated